Amplify/DevMenu/View/DetailViewFactory.swift

```diff
@@ -0,0 +1,23 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import SwiftUI
+
+/// A factory to create detail views based on `DevMenutItem`
+@available(iOS 13.0, *)
+public class DetailViewFactory {
+
+    public static func getDetailView(devMenuItemTag: Int) -> AnyView {
```

Comment:
I believe `AnyView` could be `some View`, then you don't need to wrap the views in `AnyView()` (type erase)

Reply:
I tried it with `public static func getDetailView(devMenuItemTag: Int) -> some View` but it required me to return same concrete class from all the switch statements.

Reply:
oh I see. Then `AnyView` is the way to go